ts/Extensions/Exporting/Exporting.ts

```diff
@@ -1772,7 +1772,7 @@ namespace Exporting {
                 '<svg xmlns:xlink="http://www.w3.org/1999/xlink" '
             )
             .replace(/ (|NS[0-9]+\:)href=/g, ' xlink:href=') // #3567
-            .replace(/\n/, ' ')
+            .replace(/\n/g, ' ')
```

Comment:
It might be also good to do `/\n+/g` here. Otherwise we end up with a large space replacements.